addons/sale_loyalty/tests/common.py

```diff
@@ -192,7 +193,7 @@ def _apply_promo_code(self, order, code, no_reward_fail=True):
             raise ValidationError(status['error'])
         if not status and no_reward_fail:
             # Can happen if global discount got filtered out in `_get_claimable_rewards`
-            raise ValidationError('No reward to claim with this coupon')
+            raise ValidationError(_('No reward to claim with this coupon'))
         coupons = self.env['loyalty.card']
```

Comment:
I don't think it's useful to translate error messages from tests ;)

Reply:
thanks and updated